logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -120,9 +146,20 @@ def report(stats)
                 :outputs => plugin_stats(stats, :outputs)
               },
               :reloads => stats[:reloads],
-              :queue => stats[:queue]
+              :queue => extended_stats["queue"]
```

Comment:
is this change intentional? why do we need to reach into `extended_stats["queue"]` for this value, if it has historically been available on `stats[:queue]`?

Reply:
This change originates from [this requirement](https://github.com/elastic/logstash/pull/10576#issuecomment-491331012).